pkg/controller/operators/catalog/operator.go

```diff
@@ -1331,10 +1331,44 @@ func (o *Operator) syncInstallPlans(obj interface{}) (syncError error) {
 		return
 	}
 
+	if plan.Status.Phase == v1alpha1.InstallPlanPhaseFailed {
+		return
```

Comment:
This seemed like an obvious step to me but I'm probably missing something if it wasn't here before.

Is the InstallPlan sync ever expected to recover back from a `Failed` InstallPlan? I thought that was terminal and you have to start over with a new one.